src/App.js

```diff
@@ -90,7 +102,7 @@ class App extends React.Component {
   }
 
   componentDidMount = () => {
-    if(localStorage.getItem('wca') !== null){
+    if(localStorage.getItem('fprint') !== null){
```

Comment:
"fprint" doesn't seem to be a better name for a saved token. Call it "session", or "jwt" instead.
PS: Single quotes should be the default in ES6.

Reply:
Well, actually it's just a sad and low-level try to hide what this thing is. ಥ_ಥ

Reply:
lmao xD